frontend/src/components/EditableField.js

```diff
@@ -103,7 +103,7 @@ export class EditableField extends React.Component<{
         if (!this.state.editing) {
             return (
                 <a className='editable editable-click' onClick={this.onClick}>
-                    {this.state.value === '' ? '<none>' : this.state.value}
+                    {this.state.value === '' ? 'untitled' : this.state.value}
```

Comment:
'untitled' doesn't quite make sense for general fields...(oh, that's a problem too?)
Maybe '(empty)'?
I'd put parentheses to make it look not like the actual value.

Reply:
Whatever we choose, we need to be consistent. We use `<none>` for editable fields too (https://github.com/codalab/codalab-worksheets/blob/5b9390b5ecdead2ef736c4e43dc94037f2aeb161/frontend/src/components/EditableField.js#L137) and for empty files (https://github.com/codalab/codalab-worksheets/blob/ce7a845567055f0f5ac262fe1498c4ac5cdc79be/codalab/lib/formatting.py#L14) and we don't use "(empty)" anywhere else in the codebase.

I would suggest keeping this to `<none>` and then perhaps changing the default placeholder in a separate PR if we want to do that?